resources/metainfo/wire-desktop.appdata.xml

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>wire-desktop</id>
```

Comment:
So what would be the [correct ID](https://www.freedesktop.org/software/appstream/docs/chap-Metadata.html#tag-id-generic) for Wire?

- `wire-desktop`
- `com.wire.wire-desktop`
- `com.wireapp.wire-desktop`
- `com.wire.WireDesktop`

...or some other? What I've seen is that it's usually in this form: `tld.vendor.product` – but those can vary.